src/wp-content/themes/twentytwentyone/inc/template-functions.php

```diff
@@ -388,11 +388,10 @@ function twenty_twenty_one_print_first_instance_of_block( $block_name, $content
  *
  * @since Twenty Twenty-One 1.0
  *
- * @param int|WP_Post $post Optional. Post ID or WP_Post object. Default is global $post.
```

Comment:
Add doc for **global $post;**

* @global WP_Post $post Global post object.

Reply:
Done